module/programs/helper.js

```diff
@@ -103,9 +104,19 @@ module.exports = class ProgramsHelper {
               "quality" : 10
           },
           "components" : [],
-          "isAPrivateProgram" : data.isAPrivateProgram ? data.isAPrivateProgram : false  
+          
         }
-        
+        _.assign(programData, {
```

Comment:
@ankitpws consider using https://www.freecodecamp.org/news/three-dots-operator-in-javascript/, basically for any new field added in the model tomorrow, should be readily available in create API without doing any code change. We should validate the key from request is defined in model.

Reply:
used three dot object assign method